cargo-shuttle/src/init.rs

```diff
@@ -0,0 +1,342 @@
+use std::fmt;
+use std::fs::{read_to_string, File};
+use std::io::Write;
+use std::path::PathBuf;
+
+use anyhow::Result;
+use cargo::ops::NewOptions;
+use cargo_edit::{find, get_latest_dependency, registry_url};
+use crate::args::InitArgs;
+use indoc::indoc;
+use toml_edit::{value, Array, Document, Item, Table};
+use url::Url;
+
+
+#[derive(Debug, Copy, Clone, PartialEq)]
+pub enum Framework {
+    Axum,
+    Rocket,
+    Tide,
+    Tower,
+    Default,
```

Comment:
What will the idea with `Default` be? Will it still generate a template, or just error?

Reply:
Thanks for asking. I think it just won't generate anything. I used it as a condition. If this enum option is not necessary, do you have any suggestion to handle when user doesn't specify a flag? 🙏